W-18662606 ch2 https http lds (#801)
mulesoft/docs-runtime-manager: cloudhub/modules/ROOT/pages/app-migration.adoc

When sending traffic via SLB in CloudHub applications, you can target either HTTP or HTTPS. This impacts on whether the SLB uses port 8081 or 8082 to forward requests to your application: 

* If you target HTTP, the SLB receives traffic on HTTP and forwards it to port 8081. The app then receives on port 8081.
* If you target HTTPS, the SLB receives traffic on HTTPS and forwards it to port 8082. The app then receives on port 8082.

CloudHub 2.0 private spaces are configured by default to redirect traffic to HTTPS. If you use a mix of HTTP and HTTPS applications in CloudHub SLB and want to migrate this configuration, use either of these workarounds:

* Enable the *Accept HTTP* option within the *Advanced* tab of the private space settings. This allows CloudHub 2.0 to accept HTTP traffic. 
+
image::accept-http.png[Option to accept HTTP traffic.]
+
[WARNING]
This option applies to all client-to-private-space traffic. If you enable it, all the applications in that private space become open to accepting HTTP traffic.
+
* Update your CloudHub HTTP applications to become HTTPS. In this case, you can leave the “redirect to HTTPS” option on the private space settings. 
+
[NOTE]

> This workaround can take more work and time depending on how many HTTP CloudHub applications you have.

Reviewer: ```suggestion
This workaround can take more work and time depending on the number of HTTP CloudHub applications you have.
```
